apps/aetestframework/mix.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Aetestframework.MixProject do
+  use Mix.Project
+
+  def project do
+    [
```

Comment:
are all of these needed? and equal to the other mixfiles?

Reply:
Yes, because it's a separate app and I can't remove the mix file as it's a part of the umbrella project structure

Reply:
the other apps specify way less things here

Reply:
e.g. https://github.com/aeternity/elixir-node/blob/master/apps/aeutil/mix.exs